source/includes/gcp/_forwarding_rules.md

```diff
@@ -1,9 +1,150 @@
 #### Forwarding rules
 
 A forwarding rule and its corresponding IP address represent the frontend configuration of a load balancer.
+
+<!-------------------- LIST FORWARDING RULE -------------------->
+##### List forwarding rules
+
+```shell
+curl -X GET \
+  -H 'MC-Api-key: your_api_key'
+  "https://cloudmc_endpoint/v1/services/gcp/test-area/forwardingrules"
+  # The above command returns JSON structured like this:
+```
+
+```json
+{
+  "data": [
+    {
+      "creationTimestamp": "2019-10-22T15:23:35.006-07:00",
+      "description": "",
+      "IPAddress": "35.227.251.25",
+      "IPProtocol": "TCP",
+      "ipVersion": "IPV4",
+      "kind": "compute#forwardingRule",
+      "loadBalancingScheme": "EXTERNAL",
+      "networkTier": "PREMIUM",
+      "portRange": "80-80",
+      "selfLink": "https://www.googleapis.com/compute/v1/projects/test-area/global/forwardingRules/forwardingrulename",
+      "target": "https://www.googleapis.com/compute/v1/projects/test-area/global/targetHttpProxies/targetproxyname",
+      "shortTarget": "targetproxyname",
+      "id": "4724455712741277576",
+      "name": "gfr-user-name"
+    }
+  ],
+  "metadata": {
+    "recordCount": 1
+  }
+}
+```
+
+<code>
+   GET /service/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/forwardingrules
+</code>
+
+Retrieve a list of all forwarding rules
+
+Attributes | &nbsp;
+------- | -----------
+`creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format.
+`description` <br/>*string* | An optional description of this resource.
+`IPAddress`<br/>*string* | The IP address that this forwarding rule serves. 
+`IPProtocol`<br/>*enum* | The IP protocol to which this rule applies. The default protocol is TCP.
+`ipVersion`<br/>*enum* | The IP version that will be used by this forwarding rule. The default IP version is IPV4.
+`kind`<br/>*string* | Type of the resource.
+`loadBalancingScheme`<br/>*enum* | Specifies the forwarding rule type. Possible values are INTERNAL and EXTERNAL.
```

Comment:
```suggestion
`loadBalancingScheme`<br/>*enum* | Specifies the forwarding rule type. EXTERNAL is the only value supported.
```

Reply:
change it to "The default value is EXTERNAL"